update.sh

```diff
@@ -9,9 +9,7 @@ repo="rust"
 branch="master"
```

Comment:
You can use these user/repo/branch variables to point the update script at your fork/branch, so that you can include the new code in here, so it's easier to see the effect of the change.

Reply:
Done!